dummy_prods/query_spiacs_lc.txt

```diff
@@ -0,0 +1,103 @@
+# IJD TIME COUNTS
```

Comment:
since other dummy products will not come from repo, it's better that these ones do not as well

Reply:
Do you have already a location we can use?
Or they can be simply added, to the ones for the other integral instruments...?

Reply:
if they're in the same archive of the other integral instrument, then this PR is not needed, rather just https://github.com/oda-hub/dispatcher-chart/pull/23

Reply:
I am a bit concerned putting full path in the package, it's rather be configurable by a variable or so. But if needed this can be changed later.